src/clj/nnts2/organization/routes.clj

```diff
@@ -0,0 +1,6 @@
+(ns nnts2.organization.routes
+  (:require [nnts2.organization.handler :as handler]
+            [compojure.core :refer [defroutes POST]]))
+
+(defroutes routes
+  (POST "/org" [] handler/create))
```

Comment:
Move context out of the organization routes into base level routes file.
Here it can be just post "/" and get "/"
This enables faster lookup of any route for debugging when the project has grown bigger.

Reply:
Since you've created a base routes file, I'll just add to it after merge. Probably in a round of refactoring.